src/Components/ImagesTable/ImageLink.js

```diff
@@ -26,20 +26,25 @@ const ProvisioningLink = ({ imageId, isExpired, isInClonesTable }) => {
   );
 
   if (!error) {
-    image = image.isClone ? parent : image;
+    const provisionedImage = image.isClone ? parent : image;
```

Comment:
This is a super nit-pick but now that we don't re-assign `image` can we make it a `const` (on line 11)?